zhaquirks/xiaomi/aqara/roller_curtain_e1.py

```diff
@@ -99,218 +245,104 @@ async def command(
-                    WindowCovering.cluster_id,
-                ],
-                OUTPUT_CLUSTERS: [
-                    Ota.cluster_id,
-                    Time.cluster_id,
-                ],
-            },
-            # <SizePrefixedSimpleDescriptor endpoint=242 profile=41440 device_type=97
-            # device_version=0,
-            # input_clusters=[]
-            # output_clusters=[33]>
-            242: {
-                PROFILE_ID: zgp.PROFILE_ID,
-                DEVICE_TYPE: zgp.DeviceType.PROXY_BASIC,
-                INPUT_CLUSTERS: [],
-                OUTPUT_CLUSTERS: [
-                    GreenPowerProxy.cluster_id,
-                ],
-            },
-        },
-    }
+        present_value: Final = ZCLAttributeDef(
+            id=0x0055, type=t.uint16_t, access="r*w", mandatory=True
+        )
 
 
-class RollerE1AQ_3(RollerE1AQ):
-    """Aqara Roller Shade Driver E1 (version 3) device."""
-
-    signature = {
-        MODELS_INFO: [(LUMI, "lumi.curtain.acn002")],
-        ENDPOINTS: {
-            # <SizePrefixedSimpleDescriptor endpoint=1 profile=260 device_type=514
-            # device_version=1
-            # input_clusters=[0, 2, 3, 4, 5, 6, 9, 13, 19, 258]
-            # output_clusters=[10, 25]>
-            1: {
-                PROFILE_ID: zha.PROFILE_ID,
-                DEVICE_TYPE: zha.DeviceType.WINDOW_COVERING_DEVICE,
-                INPUT_CLUSTERS: [
-                    Alarms.cluster_id,
-                    AnalogOutput.cluster_id,
-                    Basic.cluster_id,
-                    DeviceTemperature.cluster_id,
-                    Groups.cluster_id,
-                    Identify.cluster_id,
-                    MultistateOutput.cluster_id,
-                    OnOff.cluster_id,
-                    Scenes.cluster_id,
-                    WindowCovering.cluster_id,
-                ],
-                OUTPUT_CLUSTERS: [
-                    Ota.cluster_id,
-                    Time.cluster_id,
-                ],
-            },
-            # <SizePrefixedSimpleDescriptor endpoint=242 profile=41440 device_type=97
-            # device_version=0,
-            # input_clusters=[]
-            # output_clusters=[33]>
-            242: {
-                PROFILE_ID: zgp.PROFILE_ID,
-                DEVICE_TYPE: zgp.DeviceType.PROXY_BASIC,
-                INPUT_CLUSTERS: [],
-                OUTPUT_CLUSTERS: [
-                    GreenPowerProxy.cluster_id,
-                ],
-            },
-        },
-    }
+(
+    QuirkBuilder(LUMI, "lumi.curtain.acn002")
+    # temporarily commented out due to potentially breaking existing blueprints
+    #    .friendly_name(
+    #        manufacturer="Aqara", model="Roller Shade Driver E1"
+    #    )
+    .prevent_default_entity_creation(endpoint_id=1, cluster_id=AnalogOutput.cluster_id)
+    .prevent_default_entity_creation(
+        endpoint_id=1, cluster_id=MultistateOutput.cluster_id
+    )
+    .removes(OnOff.cluster_id)
```

Comment:
For this use case, why remove the cluster entirely? I'm hoping to replace uses of `removes` with `prevent_default_entity_creation`.

Reply:
As far as I can tell this cluster is non-functional.

Interacting with the switch results in an error, so there would be no benefit exposing it.
If it's optional entity a user might be tempted to enable it and raise a bug report that the switch doesn't work.

Reply:
I think `removes(OnOff.cluster_id)` can be replaced with `prevent_default_entity_creation(cluster_id=OnOff.cluster_id)`. Removing the cluster prevent prevents automatic entity discovery, which is more explicit with `prevent_default_entity_creation` and would allow you to still interact with the `OnOff` cluster via cluster commands if necessary.

Reply:
I'd usually agree, but the device itself is responding with the below error to **any** command to OnOff:

Failed to perform the action zha/issue_zigbee_cluster_command. Failed to issue cluster command with status: <Status.UNSUPPORTED_CLUSTER: 195>

Reply:
@puddly if we're planning to deprecate `removes` then happy to change this to use `prevent_default_entity_creation`.

Reply:
I've updated the PR to use `prevent_default_entity_creation` https://github.com/zigpy/zha-device-handlers/pull/3686/commits/872442a3be297f93968ec0d9519d8cb2b33dc43f

Reply:
Hi @puddly, did you have any other feedback on these changes?

Reply:
Not at the moment, sorry. The above method depends on https://github.com/zigpy/zha/pull/365 to be finished and there were a few hiccups getting it merged.